### DISCUSSION TITLE
Question: Using the Tree component, how would I scroll to selected/expanded node?

### DISCUSSION THREAD

**Discussion by mperezInnotas** [primefaces/community#2291] [PrimeReact]
If I have a Tree component which is inside a container that's say 400px and I have a `overflow-y: scroll` on it, when I used the selectedKeys/expandedKeys props. How do I scroll to a node which has been selected/expanded when it happens to be off screen because of the `overflow-y: scroll`?  Generally I would think I'd somehow get a reference to the HTML element for that rendered node, then use a `scrollIntoView()` call to get it to scroll into view.  I'm just not sure how to know 1) when are the tree nodes actually rendered to the DOM and 2) would I use a css selector to get the element which has `p-highlighted` as a class in it?

**Comment by liborjelinek** [comment]
@mperezInnotas Did you manage it somehow?
